eo-parser/src/main/java/org/eolang/parser/errors/EoParserErrors.java

```diff
@@ -36,22 +35,15 @@
 import org.antlr.v4.runtime.Recognizer;
 import org.antlr.v4.runtime.Token;
 import org.cactoos.Text;
-import org.cactoos.iterable.Mapped;
-import org.cactoos.list.ListOf;
-import org.cactoos.text.UncheckedText;
+import org.eolang.parser.EoParser;
+import org.eolang.parser.ParsingException;
 import org.xembly.Directive;
-import org.xembly.Directives;
 
 /**
- * Accumulates all parsing errors.
- *
- * @since 0.30.0
- * @todo #3706:30min Split {@link ParsingErrors} into two classes.
- *  Currently we use the same {@link ParsingErrors} class to accumulate all the parsing errors
- *  despite their origin. This class should be split into two classes: one for parsing errors
- *  {@link ParserErrors} and another for lexer errors {@link LexerErrors}.
+ * Accumulates all parsing errors related to EO parser.
+ * @since 0.50
  */
-final class ParsingErrors extends BaseErrorListener implements Iterable<Directive> {
+public final class EoParserErrors extends BaseErrorListener implements Iterable<Directive> {
```

Comment:
@volodya-lombrozo same here, we have `Dr` prefix for class that are iterables of `Directive`. Maybe can use it too?

Reply:
@yegor256 Renamed to `DrEoParserErrors`